src/typeMapper.js

```diff
@@ -1,12 +1,22 @@
 import {
-  GraphQLInt,
-  GraphQLString,
+   GraphQLInt,
+   GraphQLString,
    GraphQLBoolean,
    GraphQLFloat,
    GraphQLEnumType,
    GraphQLList
  } from 'graphql';
 
+let _userMappingFunc;
+/**
+ * A function to set a custom mapping of types
+ * @param mapFunc
+ */
+export const mapType = (mapFunc = () => false) => {
+    _userMappingFunc = mapFunc;
+};
```

Comment:
My only reasoning for this was lower in the file, when we use their function, we don't run into errors when we do `_userMappingFunc(sequelizeType)`. The alternative would be set the _userMappingFunc to a function in the declaration. I guess that still doesn't solve someone passing an object or something.. I can take it out and type check it before I call it to be extra sure.